README.md

```diff
@@ -46,17 +46,32 @@ When using such data management platform to overcome the block disk device limit
 - capacity management and monitoring
 - resizing and optimizing layouts related to capacity management
 - decommissioning the devices in secure way
+  - by default every resource created by DiskoBlock has a finalizer, so deletion will be blocked until coresponding DiskConfig is exists
```

Comment:
> by default every resource created by DiskoBlock has a finalizer, so deletion will be blocked until coresponding DiskConfig is exists

I'm a bit confused by this, will deletion be blocked if `DiskConfig` exists or is it blocked until `DiskConfig` is removed, this comment suggests the first option

Reply:
Also `corresponding` instead of `coresponding` and I believe `DiskoBlock` is meant to be `DiscoBlocks`, but I may be wrong \:)

Reply:
@Mojachieee Could you please suggest a short sentence? When you delete DiskConfig, DiscoBlocks removes finalizers of every object. So you should delete them. Otherwise, finalizers holding deletion.

Reply:
```
by default every resource created by DiscoBlocks has a finalizer, so deletion will be blocked until the corresponding DiskConfig has been deleted
```